workflow/controller/operator_test.go

```diff
@@ -8510,15 +8512,20 @@ func TestMutexWfPendingWithNoPod(t *testing.T) {
 	cancel, controller := newController(wf)
 	defer cancel()
 	ctx := context.Background()
-	controller.syncManager = sync.NewLockManager(GetSyncLimitFunc(ctx, controller.kubeclientset), func(key string) {
```

Comment:
Why has this been removed in this iteration?

Reply:
syncManager will be created by `newController`, so maybe this is unnecessary

https://github.com/argoproj/argo-workflows/blob/5310c396ee1e9f4e5575f62ef5340a86540d8679/workflow/controller/controller_test.go#L331-L333

Reply:
thanks @Joibel , i got it. 
done